app/services/reject_application.rb

```diff
@@ -1,17 +1,20 @@
 class RejectApplication
-  Response = Struct.new(:successful?, :application_choice)
+  include ActiveModel::Validations
 
   def initialize(application_choice:, rejection:)
     @application_choice = application_choice
     @rejection = rejection
   end
 
-  def call
+  def save
     ApplicationStateChange.new(@application_choice).reject_application!
```

Comment:
Should this and the below update be wrapped in a transaction?

Reply:
Done